### PR TITLE
[Keymap] Add ddone's keymap

### PR DESCRIPTION
**Pull request by Deedone**
## Description
My keymap

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/iris/keymaps/ddone/keymap.c [hunk 1]
```diff
@@ -0,0 +1,145 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/keebio/iris/keymaps/ddone/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/keebio/iris/keymaps/ddone/keymap.c [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 3
```
Comment (drashna) [inline keyboards/keebio/iris/keymaps/ddone/keymap.c:6]: We highly recommend using enum's over defines, as it's easier to maintain long term.  It starts at 0 and auto-increments. 
```suggestion
enum custom_layers {
    _QWERTY,
    _LOWER,
    _RAISE,
    _ADJUST,
};
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/keebio/iris/keymaps/ddone/keymap.c [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+                                    _______, _______, _______,                   _______, _______, _______
+                                // └────────┴────────┴────────┘                 └────────┴────────┴────────┘
+  )
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+        set_single_persistent_default_layer(_QWERTY);
```
Comment (drashna) [inline keyboards/keebio/iris/keymaps/ddone/keymap.c:99]: You don't need this macro if you only have the one default layout

File: keyboards/keebio/iris/keymaps/ddone/rules.mk [hunk 4]
```diff
@@ -0,0 +1,4 @@
+LTO_ENABLE = yes
+COMMAND_ENABLE = no
+MOUSEKEY_ENABLE = yes
+BOOTMAGIC_ENABLE = no
```
Comment (drashna) [inline keyboards/keebio/iris/keymaps/ddone/rules.mk:4]: I would recommend: 
```suggestion
BOOTMAGIC_ENABLE = lite
```

This has none of the annoying hold key stuff, but can/will allow yo to hold a key to get into the bootloader.

**Comment by Deedone** [comment]
Thanks for suggestions

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks for the contribution!

**Pull request merged**